ros/lib/config.py

```diff
@@ -25,3 +26,16 @@
 APP_NAME = os.getenv("APP_NAME", "ros")
 
 INSIGHTS_EXTRACT_LOGLEVEL = os.getenv("INSIGHTS_EXTRACT_LOGLEVEL", "ERROR")
+
+# Time interval after which garbage collector is involved to check for outdated data.
+GARBAGE_COLLECTION_INTERVAL = os.getenv("GARBAGE_COLLECTION_INTERVAL", 86400)
+# Number of days after which data is considered to be outdated.
+DAYS_DATA_OUTDATED = os.getenv("DAYS_DATA_OUTDATED", 14)
```

Comment:
Will it be good idea to rename this variable to something like -
 `METRICS_PURGE_DAYS` or `DAYS_UNTIL_STALE` ?

Reply:
Why 14, if already deleting data older than 10 days?

Reply:
@apuntamb We have decided to delete the data older than 14 days to make it consistent with host inventory.

Reply:
Sounds good, can we modify the title of the PR then?

Reply:
@apuntamb, any thoughts on renaming this variable or should I keep as it is?

Reply:
How about `DAYS_SINCE_OUTDATED` or `DAYS_SINCE_STALE` ?